project/Win32BuildSetup/BuildSetup.bat

```diff
@@ -56,61 +55,18 @@ FOR %%b in (%1, %2, %3, %4, %5, %6) DO (
   IF %%b==nomingwlibs SET buildmingwlibs=false
   IF %%b==nobinaryaddons SET buildbinaryaddons=false
   IF %%b==sh SET useshell=sh
-  IF %%b==cmake SET cmake=1
 )
 
 SET buildconfig=Release
-set WORKSPACE=%CD%\..\..
+set WORKSPACE=%CD%\..\..\kodi-build
```

Comment:
Yeah I was on my way out the door so I just pushed, cleaning it up and doing it properly today